composer.json

```diff
@@ -9,5 +9,34 @@
         "php": ">=5.3.3",
         "zendframework/zendframework": "2.2.*",
         "zendframework/zendservice-recaptcha": "2.0.*"
+    },
+    "autoload": {
+        "psr-4": {
+            "Api\\": "./module/Api",
+            "Application\\": "./module/Application/src/Application",
```

Comment:
ah yes yes :) would you send me a PR?

2015-02-21 19:50 GMT+01:00 Luis Cordova <notifications@github.com>:

> In composer.json
> <https://github.com/zendframework/zf-web/pull/164#discussion_r25124488>:
>
> > @@ -9,5 +9,34 @@
> >          "php": ">=5.3.3",
> >          "zendframework/zendframework": "2.2.*",
> >          "zendframework/zendservice-recaptcha": "2.0.*"
> > +    },
> > +    "autoload": {
> > +        "psr-4": {
> > +            "Api\\": "./module/Api",
> > +            "Application\\": "./module/Application/src/Application",
>
> why the dots? i think all can go away? [image: :older_man:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendframework/zf-web/pull/164/files#r25124488>.
>



-- 
Gianluca Arbezzano
www.gianarb.it